### DISCUSSION TITLE
Help me.

### DISCUSSION THREAD

**Discussion by Chousik** [community/community#51337] [GitHub Education]
6

**Comment by samus-aran** [comment]
Hi @Chousik 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by Chousik** [reply]
![image](https://user-images.githubusercontent.com/78282521/229102624-bd7a828c-3b04-4a65-b6c9-99a47ac49702.png)
